cfgBuilder/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+/*
+Builds the configuration files for the chainbridge application.
+*/
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var app = cli.NewApp()
+
+func init() {
+	app.Action = run
+	app.Copyright = "Copyright 2019 ChainSafe Systems Authors"
+	app.Name = "cfgBuilder"
+	app.Usage = "cfgBuilder [source] [destination]"
+	app.Author = "ChainSafe Systems 2019"
+	app.EnableBashCompletion = true
+}
+
+func run(ctx *cli.Context) error {
```

Comment:
Seems like `func(*cli.Context) {}` is considered deprecated right now too, and docs mention support will be removed for it in a later version. Do we want to get ahead of that?

Reply:
Not a priority here, opened issue to update to v2 over in ChainBridge